perf/src/index.tsx

```diff
@@ -85,3 +92,36 @@ window.runMeasures = async (filter: string = '') => {
 
   return performanceMeasures
 }
+
+//
+// Control tools
+//
+const Control: React.FunctionComponent = () => {
+  const [filter, setFilter] = React.useState('')
+
+  return (
+    /* Heads up! This provider also does first run work! */
```

Comment:
lets be a bit more explicit about `first run` meaning. Essentially, we need to provide the implications for the perf tests first, and then elaborate on the reason for them, something like

> On first run, this Provider increases measured time due to style DOM elements being rendered to the browser. Subsequent rerenders, in contrast, are not rendering these style DOM elements again.

Reply:
👍